providers/redhat/api/client.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/facebookincubator/nvdtools/providers/lib/download"
+	"github.com/facebookincubator/nvdtools/providers/lib/runner"
+	"github.com/facebookincubator/nvdtools/providers/redhat/schema"
+)
+
+const (
+	perPage    = 50
+	numRetries = 3
```

Comment:
I'd increase it to 5. Or even better, make that thing configurable (also for Flexera client). But the latter probably belongs to its own diff.

Reply:
Increased it to 5 for now, will at some point make all providers use the Client interface from previous pull. Now focusing only on this one